taxi/models.py

```diff
@@ -1,5 +1,6 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django.forms import forms
```

Comment:
The import `from django.forms import forms` is incorrect for model validation. You should use `from django.core.exceptions import ValidationError` instead.